sessions/recap-project-5/art-gallery-app/README.md

```diff
@@ -49,228 +43,233 @@ Download a [Favorite Icon](https://lucide.dev/icons/) and save it in an "assets"
 
 🚀 Project Deployment to Vercel is required: Please follow [these instructions](https://github.com/neuefische/web-protocol-template/blob/main/docs/deployment-vercel.md) to deploy the project on Vercel.
 
-## User Stories
+## Tasks
+
+This app will include four main pages: **Spotlight**, **Gallery**, **Favorite**, and **Detail**.
+
+Each page will need access to the information from the API. Consider where to fetch and store the art piece data to ensure that all pages can easily access it.
+
+<details>
+<summary>Hints 🙈</summary>
+In Next.js, the place to store data or states that need to be accessed across the app is in `_app.js`.
+
+It's best practice to consolidate related information in a single state. However, if you're not comfortable with this approach, you can use separate states. Keep in mind that you’ll need to identify which information corresponds to each art piece, so consider saving the slug or a unique identifier in each state to maintain clarity.
+
+</details>
+
+### 1. Spotlight Page
```

Comment:
Thanks for the suggestion! I think I understand your perspective, but I’d like to clarify a couple of points to ensure we’re aligned.

Are you suggesting that we initially have students fetch the data on the Gallery page and then move it to `__app`? Personally, I feel that having them start with fetching data in one place and then moving it later could lead to unnecessary complexity without adding much learning value. I think it’s beneficial for students to first think critically about *where* data should be fetched based on app needs rather than having them rework that setup.

However, if you’re simply suggesting that students start with building the Gallery page first, followed by the Spotlight page, I’m completely on board with that! Reordering the tasks to start with the Gallery would work well without requiring them to rework data fetching.

One thought: if we do want them to practice changing the data fetch location as a learning exercise, maybe we could create a dedicated challenge in the data-fetching session. This way, they could focus on it as a standalone concept.

Additionally, all tasks are numbered, so if it helps with clarity, we could add “User Story #” before each task to reinforce the order.

Let me know what you think!